src/hooks/screen/useCreateChat.ts

```diff
@@ -63,6 +63,7 @@ const useCreateChat = () => {
       const initChannel = await SignedMessageRepo.createSignedChat(members);
       const chatData = createChannelJson(initChannel, selectedUser);
 
+      console.log('initChannel', JSON.stringify(initChannel));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging sensitive data like the `initChannel` object can pose a security risk if it contains any private or sensitive information. It's generally not a good practice to log sensitive data, especially in production code. If this is for debugging purposes, make sure to remove it before merging.
```diff
- console.log('initChannel', JSON.stringify(initChannel));
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->